pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -47,6 +47,10 @@ export function mounting_dialog(client, block, mode, forced_options) {
     const [old_config, old_dir, old_opts, old_parents] = get_fstab_config(block, true);
     const options = old_config ? old_opts : initial_tab_options(client, block, true);
 
+    const old_dir_for_display = client.strip_mount_point_prefix(old_dir);
+    if (old_dir_for_display === false)
+        return Promise.reject(_("This device can not be used for the installation target."));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231213-115423-cbae7212-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)